opm/simulators/flow/BlackoilModel.hpp

```diff
@@ -527,8 +527,10 @@ namespace Opm {
         // compute the "relative" change of the solution between time steps
         Scalar relativeChange() const
         {
-            Scalar resultDelta = 0.0;
-            Scalar resultDenom = 0.0;
+            Scalar resultDeltaPressure = 0.0;
```

Comment:
granted a very minor optimization, but can we do
```
std::array<Scalar,4> r;
auto& [resultDeltaPressure, resultDenomPressure, resultDeltaSaturation, resultDenomSaturation] = r;
```
and then do
```
comm.summ(r.data(), 4);
```
below?

Reply:
Sounds reasonable. We would still need to initialise the Scalars to 0 though?

Reply:
which is what {} after r does.

Reply:
Ah, I see! Thanks!

Reply:
Should it be `gridView.comm.sum(r.data(), 4);`?

Reply:
indeed i took a shortcut.